src/test/java/io/strimzi/kafka/quotas/StaticQuotaCallbackTest.java

```diff
@@ -4,16 +4,6 @@
  */
 package io.strimzi.kafka.quotas;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Import order was inconsistent between files, to keep my editor happy I've made it consistent. 

Not sure if import order should be covered by checksytle rules?